src/components/ListItem.css

```diff
@@ -1,8 +1,9 @@
 .ListItem {
 	align-items: baseline;
-	display: flex;
+	display: inline;
```

Comment:
🔍 `display: inline-flex` might accomplish what you want here without getting rid of some of the benefits of flex layout.

Reply:
Good call. I'll make that change!